### ISSUE TITLE
Ampersands in sidebar text in Qt5 build.

### ISSUE THREAD

**Issue by Kishore96in** [clementine-player/Clementine#6118]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [ ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/) **NOTE:** Official build was not available for Arch, but I built it using the clementine-qt5-git AUR package.

### System information
- Operating System: Arch Linux
- Clementine version: 1.3.1-565-gd20c2244a (Qt5 branch, built from [this AUR package](https://aur.archlinux.org/packages/clementine-qt5-git/)).

### Expected behaviour / actual behaviour
![screenshot_20180317_003353](https://user-images.githubusercontent.com/23440033/44084979-d30cc1b8-9fd5-11e8-862e-91b006d3c223.png)
As seen in the above image, ampersands appear in the text in the sidebar. This happens in the sidebar modes 'large sidebar', 'small sidebar', and 'tabs on top', but not in the other modes. 

In the latest qt4 release (1.3.1), this issue isn't present.

### Steps to reproduce the problem (only for bugs)
1. Build Clementine from the Qt5 branch.
2. Launch Clementine.
3. Set the sidebar mode to 'large sidebar' by right-clicking on it.

**Comment by Kishore96in** [comment]
Yes, I am using KDE Plasma. The workaround in the bug you linked works. Should this be marked as duplicate, then?

**Comment by EGD** [comment]
Related bug fixed in KDE: https://bugs.kde.org/show_bug.cgi?id=337491
